python_modules/libraries/dagster-embedded-elt/dagster_embedded_elt/sling/resources.py

```diff
@@ -337,6 +341,7 @@ def replicate(
             temp_dir = tempfile.gettempdir()
             temp_file = os.path.join(temp_dir, f"sling-replication-{uid}.json")
             env = os.environ.copy()
+            logger.info(f"env: {env}")
```

Comment:
Is it possible that this will log sensitive information?

Reply:
oops this shouldn’t be here !